migrations-shelley/1573472706108_add-known-legacy-addresses-table.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable camelcase */
+
+const fs = require('fs');
+const readline = require('readline');
+const SqlString = require('sqlstring');
+
+exports.shorthands = undefined;
+
+const SQL = `
+CREATE TABLE known_legacy_addresses  (
+  address text,
+  PRIMARY KEY (address)
+);
+`
+
+exports.up = async (pgm, run) => {
+
+  pgm.sql(SQL)
+
+  const fileStream = fs.createReadStream('migrations-shelley/data/known-legacy-addresses-byron-mainnet.txt')
+  const rl = readline.createInterface({
+    input: fileStream,
+    crlfDelay: Infinity
+  })
+
+  for await (const line of rl) {
+    // ignore comments
+    if (line && !line.startsWith('#')) {
```

Comment:
Maybe it will be good to ignore spaces here:
```
const strippedLine = line.trim()
if (strippedLine && !strippedLine.startsWith('#')) {
  pgm.sql('insert into known_legacy_addresses (address) values ({line})', { line: SqlString.escape(strippedLine) })
}
```

Reply:
Defo. Good catch! I was under a wrong impression that a blank line supposed to be falsy.